Gaming (#518)
solana-developers/solana-cookbook: docs/gaming/store-sol-in-pda.md

---

> title: Storing Sol in a program

Reviewer: ```suggestion
title: Storing SOL in a PDA
```

head:
  - - meta
    - name: title

>       content: Solana Cookbook | Storing Sol in a program

Reviewer: ```suggestion
      content: Solana Cookbook | Storing SOL in a PDA
```

  - - meta
    - name: og:title

>       content: Solana Cookbook | Storing Sol in a program

Reviewer: ```suggestion
      content: Solana Cookbook | Storing SOL in a PDA
```

  - - meta
    - name: description

>       content: Show how to pay out SOL to the player from a chest vault

Reviewer: ```suggestion
      content: Using PDAs, you can reward SOL to players playing your game. Learn how to reward SOL from a PDA when players find chests in this game.
```

  - - meta
    - name: og:description

>       content: Show how to pay out SOL to the player from a chest vault

Reviewer: ```suggestion
      content: Using PDAs, you can reward SOL to players playing your game. Learn how to reward SOL from a PDA when players find chests in this game.
```

  - - meta
    - name: og:image
      content: https://solanacookbook.com/cookbook-sharing-card.png
  - - meta
    - name: og:image:alt
      content: Solana splash card
  - - meta
    - name: twitter:card
      content: summary
  - - meta
    - name: twitter:site
      content: "@solanacookbook"
  - - meta
    - name: twitter:image
      content: "https://solanacookbook.com/cookbook-sharing-card.png"
  - - meta
    - name: robots
      content: index,follow,noodp
  - - meta
    - name: googlebot
      content: index,follow
footer: MIT Licensed
---

> # Storing SOL in a Program Account and Distributing Rewards to Players

Reviewer: ```suggestion
# Storing SOL in PDAs for Game Rewards
```
